README.md

```diff
@@ -3,46 +3,85 @@
 [![Tests](https://github.com/wmo-im/csv2bufr/workflows/tests%20%E2%9A%99%EF%B8%8F/badge.svg)](https://github.com/wmo-im/csv2bufr/actions/workflows/tests.yml)
 
 csv2bufr is a Python package to transform CSV data into WMO BUFR.  Currently, csv2bufr converts each row in a 
-CSV to a BUFR message.
+CSV to a BUFR message. Both a python module and CLI to the module are provided.
 
-### Usage
+### Installation
+The csv2bufr module relies on the ECCODES software library from ECMWF. 
```

Comment:
...relies on the [ecCodes](https://confluence.ecmwf.int/display/ECC) software library....

Reply:
changed.